modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -123,7 +141,7 @@
 							<img
 								alt=${productName}
 								class="solution-search-image rounded"
-								src="${productThumbnail}"
+								src="/o/${productImageParts[1]}"
```

Comment:
What will happen if the productImageParts to be null or undefined?

Reply:
Where was productImageParts defined?

Reply:
OMG. I forgot to update this line

Reply:
And if productImageParts is null or undefined, this is put an image default there is in the portal. I took a test, where I took images of a product and it placed this default image.